src/System.IO.FileSystem/pkg/win/System.IO.FileSystem.pkgproj

```diff
@@ -11,14 +11,6 @@
     <ProjectReference Include="..\..\src\System.IO.FileSystem.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </ProjectReference>
-    <ProjectReference Include="..\..\src\System.IO.FileSystem.csproj">
```

Comment:
Ok, I have resurrected some of the netstandard1.3 / netcore50 stuff, renamed the original implementations to *.netstandard1.3.cs, and conditionally compiled those or the new implementations based on TargetGroup. Will post update shortly.